Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -340,6 +401,24 @@ bool NetworkStorageManager::removeOriginStorageManagerIfPossible(const WebCore::
     return true;
 }
 
+void NetworkStorageManager::updateOriginModificationTime(const WebCore::ClientOrigin& origin)
+{
+    assertIsCurrent(workQueue());
+    // This function should be called when origin is in use, i.e. OriginStorageManager exists.
+    auto* manager = m_originStorageManagers.get(origin);
```

Comment:
Am I correct that it is only called for each write operations, while it is only called for the first read operation during the lifetime of the network process?
This makes sense from a perf perspective, but tends to favour applications writing often compared to applications writing once and reading often.

Reply:
Not really, this is called in `writeOriginToFileIfNecessary()`, and `writeOriginToFileIfNecessary()` is called in `originStorageManager()`, and `originStorageManager()` can be called when there is storage client needs to access `OriginStorageManager` (you may search `originStorageManager()` in `NetworkStorageManager.cpp` if you want to know when it could be called).